src/auth/services/auth/auth.service.ts

```diff
@@ -28,14 +28,25 @@ export class AuthService {
 
   async login(user: UserLogin) {
     const payload = { email: user.email, id: user.id };
+    const access_token = this.jwtService.sign(payload);
+    const { password, ...userInfo } = user;
     return {
-      access_token: this.jwtService.sign(payload),
+      user: {
+        ...userInfo,
+        access_token,
+      },
```

Comment:
```suggestion
      user: {
        ...,
      },
      access_token
```
Các trường không liên quan đến model nên để ra ngoài